src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,104 @@
+        fill={
+          location.pathname === "/settings/product-management"
+            ? "white"
```

Comment:
fill={location.pathname.includes("/settings") ? "white" : "#EA7C69"}
olarak kullanılabilir settings/your-restaurant sayfasında da active olarak color'ının değişmesi için